03-objects-and-composition/exercise/06.storeCatalogue.js

```diff
@@ -0,0 +1,16 @@
+function solve(arr) {
+
+    arr.sort((a, b) => a.localeCompare(b));
```

Comment:
I would advice you to add a variable for the sorted data and not modify the passed parameter:
 `const sortedInput = arr.sort((a, b) => a.localeCompare(b));`

You can read this topic about it: https://medium.com/the-andela-way/its-bad-practice-to-reassign-function-params-c37f289a5dc4